djpymemcache/client.py

```diff
@@ -30,10 +30,17 @@ class Client(HashClient):
     """
 
     def __init__(self, servers, *args, **kwargs):
-        kwargs['serializer'] = python_memcache_serializer
-        kwargs['deserializer'] = python_memcache_deserializer
-        return super(Client, self).__init__(
-            _split_host_and_port(servers), *args, **kwargs)
+        kwargs.update(
```

Comment:
I think `setdefault` is better for performance issue :)
See: <https://stackoverflow.com/a/7423648>

Reply:
I agree.